internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -404,7 +411,7 @@ func (cf CloudFormation) changeRenderers(in changeRenderersInput) ([]progress.Re
 			}
 			renderer = r
 		case aws.StringValue(change.ResourceChange.ResourceType) == ecsServiceResourceType:
-			renderer = progress.ListeningECSServiceResourceRenderer(in.stackStreamer, cf.ecsClient, logicalID, description, progress.ECSServiceRendererOpts{
+			renderer = progress.ListeningECSServiceResourceRenderer(in.stackStreamer, cf.ecsClient, cf.cwClient, logicalID, description, progress.ECSServiceRendererOpts{
```

Comment:
What do you think of refactoring to have two different types as input to the constructor?
```go
// ECSServiceRendererCfg holds configuration required to initialize an ECS service renderer.
type ECSServiceRendererCfg struct 

// ECSServiceRendererOpts holds optional configuration to stylize an ECS service renderer. 
type ECSServiceRendererOpts struct

func ListeningECSServiceResourceRenderer(cfg ECSServiceRendererCfg, opts ECSServiceRendererOpts)
```